<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
border-radius：5px;</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>227</width>
     <height>144</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <property name="styleSheet">
       <string notr="true">border: 0px solid rgb(0,0,0);
	</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;

	border: 0px solid rgb(0,0,0);
	</string>
         </property>
         <property name="text">
          <string>服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditServerAddr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">border: 0px solid rgb(0,0,0);
	</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;

	border: 0px solid rgb(0,0,0);
	</string>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUserName">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="styleSheet">
       <string notr="true">border: 0px solid rgb(0,0,0);
	</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
border: 0px solid rgb(0,0,0);
	</string>
         </property>
         <property name="text">
          <string>  密码 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="styleSheet">
       <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonRegister">
      <property name="styleSheet">
       <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
